README.md

```diff
@@ -51,6 +51,10 @@ ServerImagekit imagekit = new ServerImagekit(publicKey, privateKey, urlEndPoint,
 
 **Note**: You can get the `apiKey`, `apiSecret`, and ImagekitId from your [Imagekit.io dashboard](https://imagekit.io/dashboard).
 
```

Comment:
The SDK readme needs to be updated to include all the new methods and code snippets. Including new `SearchQuery` parameter in the list files function.